vertexai/chat/chat.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
These samples don't have `main` as opposed to Hussain's samples; should we have consistency?

Reply:
The samples style guide (https://googlecloudplatform.github.io/samples-style-guide/#no-cli) discourages standalone executables - for maintenance reasons, AIUI.

Hussain's samples are a decent compromise, but we've not had time to discuss with other samples folks about this pattern.